litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx

```diff
@@ -321,7 +321,7 @@ const TableData: React.FC<TableDataProps> = ({ data, alertStateHandler }) => {
                 className={classes.btnImg}
               />
               <Typography
-                data-cy="configureDashboard"
```

Comment:
No need to change here, this file is for dashboard.

Reply:
Undo that. Thanks.